src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age > 0 ? <p className="Person__age">I am {person.age}</p> : ''}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {person.partnerName} is my {person.sex === `m` ? 'wife' : 'husband'}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
+  </section>
+);
+
+export const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+export const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+export const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
```

Comment:
These objects should be returned to where they were, so it will be more readable
<img width="465" alt="image" src="https://github.com/user-attachments/assets/a0f1ab63-5d23-4608-84f2-d34d985777fd">